routemaster/exit_conditions/program.py

```diff
@@ -11,7 +11,7 @@
     format_parse_error_message,
 )
 
-if False:  # typing
+if typing.TYPE_CHECKING:
```

Comment:
Do you mind splitting out all the imports from the typing module? That's how we do it elsewhere in the codebase so would be good to be consistent.

Reply:
Sure, I'll put up a separate PR for that after merging this.